devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -57,30 +64,42 @@ metadata {
 	}
 }
 
+// Called when an instance of the app is installed. Typically subscribes to Events from the configured devices and creates any scheduled jobs.
```

Comment:
Adding this comment isn't crucial, as well as...

Reply:
Removed